### DISCUSSION TITLE
`.lazy` with `type="month"` fires after single keyboard press

### DISCUSSION THREAD

**Discussion by tyler36** [livewire/livewire#4945] [7. Bugs (archived)]
I think there is an issue with using `.lazy` with `input[type=month]` in Chrome desktop

I have a component that recalculates values based on a selected month.

The component uses the following HTML code with the livewire component uses a `updatedTargetMonth` public method.
```html
<input wire:model="target_month" type="month" />
```

I used the GUI widget in Chrome to change the value without a problem (Ie. it fires the `updatedTargetMonth` method when I select a new month). However, if I use the keyboard to enter a value, it fires the  `updatedTargetMonth`  after a single digit is entered.

The [Lazy updating docs](https://laravel-livewire.com/docs/2.x/properties#lazy-updating) state:

> ... use the lazy directive modifier to listen for the native change event. Now, the $message property will only be updated when the user clicks away from the input field.

```html
<input wire:model.lazy="target_month" type="month"  />
```

... but  it doesn't seem to work.

If I remove `type="month"`, I can enter a complete date (2022-04) without an issue. 

```html
<input wire:model.lazy="target_month"  />
```

If I add `type="month"` back, it fires after the first digit again, "2".

I tried updating my version of Livewire from `2.7.2`, `2.9.0` & `2.10` but the problem remains. 

[CanIuse](https://caniuse.com/?search=type%3D%22month%22) says `type="month"` is currently supported on mobile devices and chromium-based browsers (global 78%).

PHP `7.2` (yes, I know; out of my control)
Livewire `2.7.2`, `2.9.0`, `2.10.0`
Chrome `101`

**Comment by mstroink** [comment]
I think this is an issue with all date ( type="date") input fields.  For now use .defer or add a .debounce or use a text input field.

**Comment by tyler36** [comment]
Close as "stale"
